homeassistant/components/zwave_js/api.py

```diff
@@ -631,14 +637,38 @@ async def websocket_node_metadata(
     }
 )
 @websocket_api.async_response
-@async_get_node
 async def websocket_node_alerts(
     hass: HomeAssistant,
     connection: ActiveConnection,
     msg: dict[str, Any],
-    node: Node,
 ) -> None:
     """Get the alerts for a Z-Wave JS node."""
+    try:
+        node = async_get_node_from_device_id(hass, msg[DEVICE_ID])
+    except ValueError as err:
+        if "can't be found" in err.args[0]:
```

Comment:
This isn't great, but we can improve it later. We should preferably raise specific exceptions so that downstream code can react specifically as needed and don't need to check arguments like here.